python/cudf/cudf/_lib/column.pyx

```diff
@@ -466,6 +473,10 @@ cdef class Column:
                     owner=mask_owner
                 )
 
+        null_count = None
+        if cv.has_nulls():
```

Comment:
Does `has_nulls()` check if there's actually null values or just if the mask is a non-null pointer?

Reply:
`nullable()` does the latter.